tlog.spec

```diff
@@ -152,20 +143,27 @@ getent passwd %{name} >/dev/null ||
 systemd-tmpfiles --create %{name}.conf >/dev/null 2>&1 || :
 %endif
 
-%postun
-/sbin/ldconfig
-
```

Comment:
This is still needed for older distributions (RHEL 6/7) and other distributions (Debian/Ubuntu, SUSE, etc.).